html/projects.html

```diff
@@ -0,0 +1,202 @@
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Project Browser</title>
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/bootstrap/css/bootstrap.min.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/bootstrap/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.min.js"></script>
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/es.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/bootstrap2.facetview.theme.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/jquery.facetview2.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/css/facetview.css">
+    <link rel="stylesheet" href="/vivo/themes/dco/css/browsers.css">
+
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
+            $('.facet-view-simple').facetview({
+                //search_url: '//dcotest.tw.rpi.edu/search/dco/project/_search',
+                search_url: '//localhost:9200/dco/project/_search',
+                page_size: 20,
+                sort: [{"_score" : {"order" : "desc"}}],
+                sharesave_link: true,
+                search_button: true,
+                default_freetext_fuzzify: "*",
+                default_facet_operator: "AND",
+                default_facet_order: "count",
+                default_facet_size: 15,
+                facets: [
+                    {'field': 'mostSpecificType', 'display': 'Type'},
+                    {'field': 'community.name.exact', 'display': 'Community'},
+                    {'field': 'portalGroup.name.exact', 'display': 'Portal Group'},
+                    {'field': 'leader.name.exact', 'size': 20, 'display': 'Leader'},
+                    {'field': 'participants.name.exact', 'size': 20, 'display': 'Participants'},
+                    {'field': 'submittedBy.name.exact', 'size': 20, 'display': 'Submitted By'},
+                    {'field': 'grant.label.exact', 'size': 20, 'display': 'Grants'},
+                    {'field': 'dateTimeInterval.startYear', 'size': 20, 'display': 'Start Year'},
+                    {'field': 'dateTimeInterval.endYear', 'size': 20, 'display': 'End Year'},
+                    {'field': 'reportingYear.year', 'size': 20, 'display': 'Reporting Year'}
+                ],
+                search_sortby: [
+                    {'display':'Title','field':'title.exact'}
+                ],
+                render_result_record: function(options, record)
+                {
+
+                    var html = "<tr><td>";
+
```

Comment:
I did some refactoring of the HTML to implement handlebars templates.  I did some implementation (in JS) of helper methods called by the template.  It should be fairly straightforward.

If you are having trouble getting handlebars to work you should be able to modify your current code to follow the same HTML structure and get the desired thumbnail layout.  Then I can help you with the templates when I get back.